### ISSUE TITLE
Format Cover Tile

### ISSUE THREAD

**Issue by apetrycki** [resoai/TileBoard#150]
I created a cover tile and the buttons don't center like the others do.  I'm sure there's a style I can use to fix this, but maybe it's something that should be fixed in the code?  Any help to fix this would be great.  My tile size is 110.
![image](https://user-images.githubusercontent.com/34962392/45918734-67583180-be59-11e8-866f-184cd811e8f8.png)

I also agree that click and hold on the tile should optionally give you a slider to adjust the cover.  That would be a really nice feature.

**Comment by resoai** [comment]
Would this make it any better?

entitySize: ENTITY_SIZES.SMALL,

**Comment by apetrycki** [comment]
Nope, I already had it:

`var CONFIG = {
        customTheme: CUSTOM_THEMES.COMPACT,
        transition: TRANSITIONS.ANIMATED_GPU,
        entitySize: ENTITY_SIZES.SMALL,
        tileSize: 110,
        tileMargin: 3,
`

**Comment by resoai** [comment]
You can easily add some css with custom.css 

Right click on the tile and choose “Inspect element” to see its classes and classes of the parent.

**Comment by apetrycki** [comment]
Thanks, with your help I found the problem.  With lights, the top position is calculated, so it looks right when you change the size.  Cover is not calculated.  You might want to add
`.item-cover {
  top: calc(50% - 45px);
}`

I also added 
`.item-cover--button {
  width: calc(33% - 15px);
}`

which makes the buttons look much better when changing size.

**Comment by apetrycki** [comment]
What it looks like now: ![image](https://user-images.githubusercontent.com/34962392/45979490-37f22200-c01d-11e8-83bb-293275b327c2.png)

**Comment by apetrycki** [comment]
It works ok.  I found that `.item-cover { top: 20%; }` works a little better.  I'm not sure why the calc doesn't quite work right.  I tried `.item-cover { top: calc(50% - 17px); }` which I thought would keep it dead center, but something is still pushing it down on the smaller 110px tile.  I tried to get the buttons to scale in height, but I couldn't find a method to change the font size relative to the tile container.

I tried to figure out how to do a pull request, but was unsuccessful.

at 160px tile
![image](https://user-images.githubusercontent.com/34962392/45988789-a26b8800-c046-11e8-8e95-bac96e882285.png)

at 110px tile
![image](https://user-images.githubusercontent.com/34962392/45988817-c29b4700-c046-11e8-8567-c04903163faa.png)
